appinventor/appengine/src/com/google/appinventor/client/youngandroid/YoungAndroidFormUpgrader.java

```diff
@@ -814,6 +876,55 @@ private static int upgradeEmailPickerProperties(Map<String, JSONValue> component
       // RequestFocus function was added (via TextBoxBase)
       srcCompVersion = 3;
     }
+    if (srcCompVersion < 4) {
+      // TextAlignment, FontBold, FontItalic, FontTypeface properties made visible in block editor.
+      srcCompVersion = 4; 
+    }
+    return srcCompVersion;
+  }
+
+  private static int upgradeEv3ColorSensorProperties(Map<String, JSONValue> componentProperties,
+                                                     int srcCompVersion) {
```

Comment:
Code style: continuing indentation should be 4 spaces (if you're using IntelliJ you can adjust this by turning off alignment of parameters). Make this change to all of the new methods in this file.

Reply:
I fixed the styling in the latest commit. I was wondering if it wouldn't be better to have code styles defined in an .editorconfig file or even have a code linter installed. This way there shouldn't be any issues with code style when making a pull request.